### PR TITLE
Update Anthropology major (ANT)

### PR DESCRIPTION
**Pull request by bcatoto**

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 1]
```diff
@@ -120,54 +151,96 @@
...
+                  "name": "Human Evolution",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "A course on Human Evolution or ANT 215 Human Adaptation or ANT 309 Forensic Anthropology or ANT 201 (beginning Fall 2020) or a cognate¹ biological course. Note: The department will accept Evolutionary Biology (EEB 309) and Life on Earth (EEB 211/MOL 211) as cognates if taken in AY 2019 to AY 2021 by students declaring the ANT concentration in Spring 2020 or earlier. These courses may be used to substitute for a required biological anthropology course offered by the Anthropology Department.",
+                  "course_list": [
+                    "ANT 215",
+                    "ANT 309",
+                    "ANT 201",
+                    "EEB 309",
+                    "EEB 211/MO"
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]: Typo here, I assume.

Comment (bcatoto) [reply tigerpath/majors_and_certificates/majors/ANT_2018.json]: I made a new commit that fixed this typo.

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 2]
```diff
@@ -120,54 +151,137 @@
               "name": "Required Courses",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json:151]: The required courses for this track [seem to have changed](https://anthropology.princeton.edu/undergraduate/tracks/medical-anthropology) over the last few days.

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 3]
```diff
@@ -120,54 +151,137 @@
...
-                "ANT 206*",
-                "ANT 300*",
-                "ANT 301*"
+              "explanation": null,
+              "req_list": [
+                {
+                  "name": "Foundational Course",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "ANT 240 Medical Anthropology or ANT 340 Medicine and the Humanities or ANT 461 Disability, Difference, and Race or another approved foundational course. ANT 235, Medical Humanities, in Spring 2018, satisfies this requirement. Students who have taken ANT 235 may also take ANT 340 and ANT 461 as elective courses, but not ANT 240, as Medical Humanities (ANT 235) is considered equivalent to Medical Anthropology (ANT 240).",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]:
```suggestion
                  "explanation": "One  Foundational Medical Anthropology course offered by the Department including: Medical Anthropology (ANT 240), Medicine and the Humanities (ANT 340), Psychological Anthropology (ANT 305), Race and Medicine (ANT 403)",
```

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 4]
```diff
@@ -120,54 +151,137 @@
...
+                  "course_list": [
+                    "ANT 240",
+                    "ANT 340",
+                    "ANT 461"
+                  ]
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]:
```suggestion
                  "course_list": [
                    "ANT 240",
                    "ANT 340",
                    "ANT 305",
                    "ANT 403"
                  ]
```

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 5]
```diff
@@ -120,54 +151,137 @@
...
+                },
+                {
+                  "name": "Human Evolution",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]:
```suggestion
                  "name": "Biological Anthropology",
```

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 6]
```diff
@@ -120,54 +151,137 @@
...
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "A course on Human Evolution or ANT 215 Human Adaptation or ANT 309 Forensic Anthropology or ANT 201 (beginning Fall 2020) or a cognate¹ biological course. Note: The department will accept Evolutionary Biology (EEB 309) and Life on Earth (EEB 211/MOL 211) as cognates if taken in AY 2019 to AY 2021 by students declaring the ANT concentration in Spring 2020 or earlier. These courses may be used to substitute for a required biological anthropology course offered by the Anthropology Department.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]:
```suggestion
                  "explanation": "One  Biological Anthropology course offered by the Department or a cognate Biological course including: ANT 201 (taken Fall 2020 or later), a course on human evolution, a course on human adaptation or forensic anthropology, an approved EEB or MOL course.\n\nNote: The department will accept Evolutionary Biology (EEB 309) and Life on Earth (EEB 211/MOL 211) as cognates if taken in AY 2019 to AY 2021 by students declaring the ANT concentration in Spring 2020 or earlier. These courses may be used to substitute for a required biological anthropology course offered by the Anthropology Department.",
```

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 7]
```diff
@@ -120,54 +151,137 @@
...
               "req_list": [
                 {
-                  "name": "Mind",
-                  "max_counted": 5,
-                  "min_needed": null,
-                  "explanation": "Courses pertaining to the mind: Psychological Anthropology (ANT 305); Mind, Body, and Bioethics in Japan and Beyond (EAS 312/ANT 312)",
-                  "course_list": [
-                    "ANT 305",
-                    "EAS 312/ANT 312"
+                  "name": "MedAnth Electives",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json:216]: It seems that the structure of the elective courses has changed too.
The department now requires two, one, and two courses, in separate categories. See [here](https://anthropology.princeton.edu/undergraduate/tracks/medical-anthropology).

File: tigerpath/majors_and_certificates/majors/ANT_2018.json [hunk 8]
```diff
@@ -184,60 +298,124 @@
...
                   ]
                 },
                 {
-                  "name": "Law",
-                  "max_counted": 6,
+                  "name": "Cognates",
+                  "max_counted": 2,
                   "min_needed": null,
-                  "explanation": "The Anthropology of Law (ANT 342); The American Family in Law and Society (ANT 207); Social Live, Social Forces (ANT 232); Labors of Consciousness: Culture, Capital, Moral Economy (ANT 417); Democracy and Ethnography in the United States (ANT 427)",
+                  "explanation": "LPE students are allowed to take two cognates (as defined under Program of Study above). Appropriate cognates for LPE might include Introduction to Microeconomics (ECO 100), a course taught in the Politics Department, a regional studies course, courses taken during study abroad and/or anthropology courses taught outside the Law, Politics, and Economics track.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/ANT_2018.json]:
```suggestion
                  "explanation": "A cognate is a course that the director of undergraduate studies has reviewed and deemed to be relevant to a student's independent work or correspond to a student's course of study (i.e., track). LPE students are allowed to take two cognates as part of their six elective courses. Appropriate cognates for LPE might include Introduction to Microeconomics (ECO 100), a course taught in the Politics Department, a regional studies course, and/or a course taken during study abroad. Proposed cognates must be approved by the department. Approval prior to enrollment is normally expected, however, retroactive approval is granted when warranted.",
```

**Review by bnehoran — CHANGES_REQUESTED** [review]
Unfortunately, it seems that the Anthropology department just updated the requirements on their [website](https://anthropology.princeton.edu/undergraduate/majoring-anthropology), with some new course lists and structures, in particular for the [Medical Anthropology Track](https://anthropology.princeton.edu/undergraduate/tracks/medical-anthropology).

**Review by bnehoran — APPROVED** [review]

**Pull request merged**